src/async_fuse/memfs/s3_node.rs

```diff
@@ -45,7 +40,7 @@ pub enum S3NodeData {
     /// Directory entry data
     Directory(BTreeMap<String, DirEntry>),
     /// File content data
-    RegFile(Arc<GlobalCache>),
+    RegFile,
```

Comment:
After dir entry refactor, the `NodeData` seems meaning less :thinking:

Reply:
Yes, the only thing it holds is the target path of symlink.